src/Universum/Unsafe.hs

```diff
@@ -15,28 +16,132 @@ foo = Unsafe.head
 -}
 
 module Universum.Unsafe
-       ( head
-       , tail
-       , init
-       , last
-       , at
-       , (!!)
-       , fromJust
-       , foldr1
-       , foldl1
-       , minimum
-       , maximum
-       , minimumBy
-       , maximumBy
-       ) where
-
-import Data.List (foldl1, foldr1, head, init, last, maximum, maximumBy, minimum, minimumBy, tail,
-                  (!!))
-import Data.Maybe (fromJust)
-
-import Universum.Base (Int)
+  ( head
```

Comment:
I see that all files use 7-space indentation for exports, let's keep it consistent throughout the repo and not change it.